stdlib/libblastrampoline_jll/src/libblastrampoline_jll

```diff
@@ -0,0 +1,46 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+## dummy stub for https://github.com/JuliaBinaryWrappers/libblastrampoline_jll.jl
```

Comment:
Should this file have a `.jl` extension?

Reply:
Yes - it is a JLL package.

Reply:
But it still doesn't have a `.jl` extension? Since nothing complains about this I guess it is never loaded? So is it even needed?

Reply:
Ah - I see what you mean. All we need is the LBT library installed. I suppose LinearAlgebra and SuiteSparse which link against LBT should perhaps declare a dependency on this JLL.

Reply:
I mean the file itself doesn't have an extension.

Reply:
Ah got it. Too many jlls and jls all over the place.